tests/test_oauth2client.py

```diff
@@ -209,21 +209,23 @@ def test_get_environment_gae_local(self):
 
   def test_get_environment_gce_production(self):
     os.environ['SERVER_SOFTWARE'] = ''
+    no_proxy_conf = urllib.request.build_opener(urllib.request.ProxyHandler({}))
     response = MockResponse({'Metadata-Flavor': 'Google'})
-    with mock.patch.object(urllib.request, 'urlopen',
+    with mock.patch.object(no_proxy_conf, 'open',
                            return_value=response,
-                           autospec=True) as urlopen:
+                           autospec=True) as open:
```

Comment:
`open` is a dangerous name here (since it overrides the builtin); maybe `no_proxy_open`?